pkg/client/core/service_domain_client.go

```diff
@@ -195,7 +196,7 @@ func (s *domainService) CreatePrimaryHostedZoneWithoutUserinput(_ context.Contex
 		return nil, err
 	}
 
-	zone.IsDelegated = true
+	zone.IsDelegated = false
```

Comment:
What does this do?

Reply:
It's a boolean indicating if the nameservers has been delegated or not. The `true` value was wrongfully assigned in the declarative cluster PR. I just gave it a value to remove the need for user input and later realized it should be false by default